master/master.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-export SPARK_MASTER_HOST=`hostname`
+if [ ! $SPARK_MASTER_HOST_OVERWRITE ]; then
+    export SPARK_MASTER_HOST=`hostname`
+fi
```

Comment:
``` suggestion
export SPARK_MASTER_HOST=${SPARK_MASTER_HOST:-`hostname`}
```

I also will support @gmouchakis comment here: https://github.com/big-data-europe/docker-spark/pull/71#issuecomment-496442614 as there is no need to have another ENV for just overriting the SPARK_MASTER_HOST value.